signing/signing.go

```diff
@@ -0,0 +1,48 @@
+package signing
+
+import (
+	"context"
+	"log"
+	"net/http"
+	"path/filepath"
+
+	"github.com/gorilla/mux"
+	"github.com/ossf/scorecard/v2/cron/data"
+)
+
+type ScorecardOutput struct {
+	SarifOutput string
+	JsonOutput  string
+}
+
+func GetResults(w http.ResponseWriter, r *http.Request) {
+	// Get params to build GCS filepath.
+	ctx := context.Background()
+	bucketURL := "gs://ossf-scorecard-results"
+	host := mux.Vars(r)["host"]
+	orgName := mux.Vars(r)["orgName"]
+	repoName := mux.Vars(r)["repoName"]
+	resultsFile := filepath.Join(host, orgName, repoName, "results.json")
+
+	// Sanitize input and log query.
+	resultsFile = filepath.Clean(resultsFile)
+	matched, err := filepath.Match("*/*/*/results.json", resultsFile)
+	if err != nil || !matched {
+		http.Error(w, "error verifying filepath format", http.StatusInternalServerError)
+		log.Println(matched, err)
+		return
+	}
+	log.Printf("Querying GCS bucket for: %s", resultsFile)
```

Comment:
code scanning has some useful feedback below. Let's create a function that truncates what we print to log to reduce DoS/log filling; and use this for everything we log